### ISSUE TITLE
Pokémon X/Y ‒ Both screen garbling bug

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#2261]
This happens when you open the menu on clothing customization session and it shows this transition effect for a second before entering the menu. Occurs only on *hardware renderer*
Related to #2246
![Pixelated screens](https://cloud.githubusercontent.com/assets/18496868/20865328/aa1ea91a-ba49-11e6-97f0-a4a5e47b0d59.png)

Nothing useful in log:
`HW.GPU <Critical> core\hw\gpu.cpp:TextureCopy:332: zero input width`
`HW.GPU <Critical> core\hw\gpu.cpp:TextureCopy:332: zero input width`

Switching to *software renderer* will playback the animation exactly like the actual hardware behaviour.

Tested this on commit 4f0f88b

**OS:** _Windows 10 build 14393.447 (codename Redstone 1)_
**GPU:** _nVidia GT 610 2GB DDR3 (Asus)_
**Driver:** _Version 375.95_
**OpenGL:** _Version 4.5_

**Comment by yuriks** [comment]
Can you upload a save file and instructions to reproduce this?

**Comment by Dragios** [comment]
[Pokémon X save file](https://github.com/Dragios/Cloud-Drive/blob/master/Pok%C3%A9mon%20X%20save%20file.zip?raw=true)

Once you loaded the game you will land in the Pokémon Center.

**Instruction to reproduce the issue:**
1. Move to the left and head straight up until you reach the location as shown in the screenshot.
![pokemon center](https://cloud.githubusercontent.com/assets/18496868/20865418/13894282-ba4c-11e6-9135-e3cedf7fb9c8.png)
2. *Press A* to enter to the clothing customization section.
3. *Press X* to open the menu

**Comment by JayFoxRox** [comment]
I'm assuming this is the same as #2246 ?
Eitherway, we should label this [gl-renderer] unless confirmed to be a [tex-fwd] issue.
As the other issue was confirmed we should probably re-label it (or add both labels so we can find it more easily)

**Comment by yuriks** [comment]
Please verify it's fixed with #2266

**Comment by Dragios** [comment]
Confirm fixed.

**Comment by Dragios** [comment]
Confirm fixed but it blinked once before enter the menu, weird.

**Comment by yuriks** [comment]
What do you mean with "it blinked once before the menu"? What blinked?

**Comment by Dragios** [comment]
The whole screen blink once transition effect occured before entering the menu rather than immediately display the menu without transition effect (like the one when exiting menu)

**Comment by yuriks** [comment]
Is it still skewed/misaligned like here? If not, then you should open a separate issue for it.

**Comment by Dragios** [comment]
Nope, that one is fixed.

**Comment by yuriks** [comment]
Fixed by #2266
